Lobby/LobbyHUD.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LobbyHUD : MonoBehaviour
```

Comment:
Lets rename the class to EOSLobbyUI? HUD as in heads up display is not a good fit in my opinion, no idea why unity in unet named their component like that.